dbfit-java/core/src/main/java/dbfit/fixture/StatementExecution.java

```diff
@@ -4,27 +4,20 @@
 
 public class StatementExecution implements AutoCloseable {
     protected PreparedStatement statement;
-    protected int returnValueInd = -1;
 
-    public StatementExecution(PreparedStatement statement, boolean clearParameters) {
+    public StatementExecution(PreparedStatement statement) {
```

Comment:
Right. I was thinking that `StatementExecution` could be a direct replacement for `PreparedStatement` and `CallableStatement` but it would need a `DBEnvironment`'s connection to do so.